lint/tests/test_file_lints.py

```diff
@@ -21,40 +24,56 @@
 def check_with_files(input_bytes):
     return {
         filename: (check_file_contents("", filename, six.BytesIO(input_bytes)), kind)
-        for (kind, filenames) in INTERESTING_FILE_NAMES.items()
-        for filename in filenames
+        for (filename, kind) in
+        (
+            (os.path.join("html", filename), kind)
+            for (kind, filenames) in INTERESTING_FILE_NAMES.items()
+            for filename in filenames
+        )
     }
 
 
 def test_trailing_whitespace():
     error_map = check_with_files(b"test; ")
 
-    for (filename, (errors, _)) in error_map.items():
-        assert errors == [("TRAILING WHITESPACE", "Whitespace at EOL", filename, 1)]
+    for (filename, (errors, kind)) in error_map.items():
+        expected = [("TRAILING WHITESPACE", "Whitespace at EOL", filename, 1)]
+        if kind == "web-strict":
```

Comment:
Why do we expect `web-strict` to behave differently to `web-lax` here? It's calling `check_file_contents` with the same args, so why would it give a different list of errors?